src/library/psitest_library.jl

```diff
@@ -60,6 +60,15 @@ function build_c_sys14(; add_forecasts, raw_data, kwargs...)
                 l,
                 PSY.Deterministic("max_active_power", forecast_data),
             )
+            PSY.add_time_series!(
+                c_sys14,
+                l,
+                PSY.SingleTimeSeries(
```

Comment:
@rodrigomha after some analysis from @GabrielKS this change introduces a problem because when `transform_single_time_series` is called during the tests it introduces a conflict with the forecast. We need to change this code to add the forecasts OR the SingleTimeSeries, not both.